mobile/android/android-emulator-services.ts

```diff
@@ -23,6 +24,7 @@ class AndroidEmulatorServices implements Mobile.IEmulatorPlatformServices {
 	private static TIMEOUT_SECONDS = 120;
 	private static UNABLE_TO_START_EMULATOR_MESSAGE = "Cannot run your app in the native emulator. Increase the timeout of the operation with the --timeout option or try to restart your adb server with 'adb kill-server' command. Alternatively, run the Android Virtual Device manager and increase the allocated RAM for the virtual device.";
 	private static RUNNING_ANDROID_EMULATOR_REGEX = /^(emulator-\d+)\s+device$/;
+	private static EMULATOR_EXITED_MESSAGE = "The native emulator has stopped unexpectedly. Verify its configuration and try again.";
```

Comment:
I've modeled the message after UNABLE_TO_START_EMULATOR_MESSAGE - should we change it as well?